### PR TITLE
Feature/feature tip

### PR DESCRIPTION
**Pull request by Pilopa**
## Proposed changes

Adds a new tile that shows a random feature tip with a link to the documentation on the scm-manager website.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog` or CHANGELOG.md is updated for plugins
- [x] Docs updated

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

**Comment by Pilopa** [comment]
Requires https://github.com/scm-manager/scm-manager/pull/1875

**Comment by eheimbuch** [comment]
Does not work

Screenshot from 2021-11-29 11-09-46

File: src/main/js/tips/MyTips.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const loginInfoLink = useIndexLink("loginInfo");
+  const disabled = useIsCategoryDisabled("mytips");
+  const { error, isLoading, data: loginInfo } = useQuery<LoginInfoResponse, Error>(
+    ["loginInfo"],
+    () => fetch(loginInfoLink!).then(response => response.json()),
+    {
+      enabled: !disabled && !!loginInfoLink,
+      refetchOnWindowFocus: false
+    }
+  );
```
Comment (eheimbuch) [inline src/main/js/tips/MyTips.tsx:30]: Move hook to own file?

**Review by eheimbuch — CHANGES_REQUESTED** [review]

**Comment by Pilopa** [comment]
Requires https://github.com/scm-manager/scm-manager/pull/1877

**Pull request merged**